### DISCUSSION TITLE
Can't link a table with 1 record to a table with 10000 records

### DISCUSSION THREAD

**Discussion by IvanVakhatov** [moj-analytical-services/splink#937] [Q&A] [answered]
Hello everyone!
So, I have a dataframe of size (10000, 32) - "golden" records. I want to build a real-time automapping system using splink. I take a dataframe of size (1, 32) that contains the current "test" record and I want to find a similar record from the first dataset.

Moreover, when I try to run: **_linker.estimate_u_using_random_sampling(target_rows=1e6)_**
I get an error:
----- Estimating u probabilities using random sampling -----
u probability not trained for Address - Exact match (comparison vector value: 3). This usually means the comparison level was never observed in the training data.
u probability not trained for Name - Exact match (comparison vector value: 3). This usually means the comparison level was never observed in the training data.

The main columns for comparisons are: Address, Latitude, Longitude, CompanyName, CustomerName, CustomerType, ReferenceType, TIN and so on.
Well, I wonder how can I solve this using splink? Do I have to train the model every time I get a new record? How do I train it with only one record for linking?

Thanks a lot!

**Comment by aalexandersson** [comment, marked as answer]
Did you look at the documented example [Real Time Record Linkage](https://moj-analytical-services.github.io/splink/demos/example_real_time_record_linkage.html)?

Step 1 is to load a pre-trained linkage model. 
Step 2 is to compare two records, see the `linker.compare_two_records` function.
